### ISSUE TITLE
Rebuilding Windows Runtime Component

### ISSUE THREAD

**Issue by petrejonn** [MicrosoftDocs/windows-uwp#1664]
When I build a Windows Runtime Component and try rebuilding it, i get errors "c:\dev\c++\bankaccountwrc\bankaccountwrc\bankaccount.h(2): fatal error C1083: Cannot open include file: 'BankAccount.g.h': No such file or directory". After Looking at the "Generated Files" folder, I discovered that Visual studio deletes the generated files during the "clean" operation. And it seems like the files in  "Generated Files"  can only be generated if its the first time of building the project. 
Any help on how to handle this will be very much appreciated.
I am using Latest version of Visual studio 17 and latest version of windows 10 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 565e1bd5-9e15-fc88-8ccc-05d09cbde209
* Version Independent ID: 63b7ce5c-d2e7-d742-4fbd-b9729595a797
* Content: [Author events in C++/WinRT - Windows UWP applications](https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/author-events#feedback)
* Content Source: [windows-apps-src/cpp-and-winrt-apis/author-events.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/cpp-and-winrt-apis/author-events.md)
* Product: **uwp**
* Technology: **platform**
* GitHub Login: @stevewhims
* Microsoft Alias: **stwhi**

**Comment by stevewhims** [comment]
Hi, @petrejonn, I'm unable to reproduce the issue. First, I followed the sections of the topic that create the finished Windows Runtime Component (it didn't seem necessary to create the consuming app for these attempted repros). I tried first with Visual Studio 2019 and didn't have a problem. Then I switched to Visual Studio 2017 (v15.9.12), NuGet (v2.0.190506.1), VSIX (v2.0.190506.1), SDK (v10.0.18362.0). I rebuilt both project and solution. Then repeated that same process each time after a) closing and reopening VS, b) using VS's Clean command, c) manually deleting all build, temporary, and generated folders. For me, everything I needed was rebuild/regenerated, as expected, each time. I tried x86, x64, and ARM configurations. ~Steve
